### DISCUSSION TITLE
Get the Node Status while it is being created

### DISCUSSION THREAD

**Discussion by bthshariff** [kubernetes-client/csharp#1320] [Q&A]
I am trying to retrieve the Kubernetes Node Status While it is being created.

```
var node = client.CoreV1.ListNode().Items.First();
foreach(var nodeStatus in node.Status.Conditions) {
    Console.WriteLine("{0} - {1}", nodeStatus.Type, nodeStatus.Status);
}
```
It was displaying the node type as : "Ready" and status as: "True". But I also want to know when the node is being created. The above foreach loop is not displaying the node status while it is being Created. Any help would be appreciated. Thanks.

**Comment by tg123** [comment]
you can check creation time in meta

**Reply by bthshariff** [reply]
Sorry, you mean node.metadata? In which property will we get the node creating status. Thank you.

**Comment by bthshariff** [comment]
getting error message: k8s.Autorest.HttpOperationException: Operation returned an invalid status code 'Forbidden'
when accessing nodeItem.Metadata.CreationTimestamp
